src/main/java/teammates/logic/core/StudentsLogic.java

```diff
@@ -628,20 +620,30 @@ private StudentEnrollDetails enrollStudent(StudentAttributes validStudentAttribu
         return studentList;
     }
 
-    private List<String> getInvalidityInfoInDuplicatedEmail(String email,
-            List<String> studentEmailList, String[] linesArray) {
-        List<String> info = new ArrayList<>();
-        info.add("Same email address as the student in line \"" + linesArray[studentEmailList.indexOf(email) + 1] + "\"");
-        return info;
+    /**
+     * Returns a {@code String} containing the invalid information of the {@code student}
+     * and the corresponding sanitized invalid {@code userInput}.
+     */
+    private String invalidStudentInfo(StudentAttributes student, String userInput) {
```

Comment:
Can rearrange the parameters to `invalidStudentInfo(String userInput, StudentAttributes student)` for consistency with `duplicateEmailInfo` and `enrollExceptionInfo`?

Reply:
Done